frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/import-providers-tab/providers/vm-import-password.tsx

```diff
@@ -86,6 +87,14 @@ const stateToProps = (state, { wizardReduxID, provider }) => {
       provider,
       OvirtProviderField.USERNAME,
       VMWareProviderField.USERNAME,
+    ) &&
+    !iGetProviderFieldAttribute(
```

Comment:
we have to check the validation type. It should fail for error/trivial error but pass for warn/info